jobs/loghost_alerts/templates/loghost.alerts.yml

```diff
@@ -24,15 +24,15 @@ groups:
             - contact Cloud Foundry administrator team
       - alert: LoghostNotEnoughSources
         expr: |
-          count(sum(loghost_total) by (director, instance)) != <%= p('loghost_alerts.nologs.directors').length() %>
+          count by (instance) (sum(loghost_total) by (director, instance)) != <%= p('loghost_alerts.nologs.directors').length() %>
```

Comment:
I've trouble understanding the point of having the the instance IP in the alert labels
It won't tell which log source is missing, it will show the IP of the vm running the exporter (here the loghost IP)
However I see no harm in adding this label, just opening the discussion to get the underneath goal

Reply:
Because the `intance` label is not present in the result, the resulting in a confusing warning message:
> LoghostNotEnoughSources (firing) : Logs are not concetrated and locally stored on loghost instance ``

As discussed offline, it might be preferable to add more relevant informations (which would need to patch the exporter), in the meantime, just fix the typos and this message.